demo/src/screens/ExampleScreenPresenter.js

```diff
@@ -14,16 +14,26 @@ export function renderBooleanOption(title, key) {
   );
 }
 
-export function renderRadioGroup(title, key, options) {
+export function renderRadioGroup(title, key, options, isRow) {
```

Comment:
can you make the last argument an object in case we want to pass other stuff in the future. 
```suggestion
export function renderRadioGroup(title, key, options, {isRow} = {}) {
```